creature.cpp

```diff
@@ -1144,6 +1144,8 @@ static string getAttackParam(AttackType type) {
     case AttackType::SPELL: return "spell";
     case AttackType::POSSESS: return "touch";
   }
+  assert("__FUNCTION__ case unknown attack type");
+  return "";
```

Comment:
You might have to rm stdafx.h.gch

On Fri, Apr 24, 2015 at 6:50 PM, moozaad <notifications@github.com> wrote:

> In creature.cpp
> <https://github.com/miki151/keeperrl/pull/674#discussion_r29064884>:
>
> > @@ -1144,6 +1144,8 @@ static string getAttackParam(AttackType type) {
> >      case AttackType::SPELL: return "spell";
> >      case AttackType::POSSESS: return "touch";
> >    }
> > +  assert("__FUNCTION__ case unknown attack type");
> > +  return "";
>
> Yeh I tried that before I commented. It gave me a huge cascade errors.
> I'll come back to it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/miki151/keeperrl/pull/674/files#r29064884>.
>